example/scaffold/pkg/server/server.go

```diff
@@ -8,8 +8,8 @@ import (
 
 	log "github.com/Sirupsen/logrus"
 	google_protobuf "github.com/golang/protobuf/ptypes/empty"
-	"github.com/radu-matei/kube-toolkit/pkg/rpc"
-	"github.com/radu-matei/kube-toolkit/pkg/version"
+	"github.com/radu-matei/kube-toolkit/example/scaffold/pkg/rpc"
+	"github.com/radu-matei/kube-toolkit/example/scaffold/pkg/version"
```

Comment:
`ktk init` or maybe `ktk create $project` would generate a project from the templates under `example/scaffold`.

It will write all the files into the CWD and then replace every occurrence of `github.com/radu-matei/kube-toolkit/example/scaffold` to your gopkg name induced from the CWD.

For example, `cd $GOPATH/src/github.com/mumoshu && ktk init myproj` would create `$GOPATH/src/github.com/mumoshu/myproj`. The contents of the directory would be the same as `example/scaffold`, but all the occurences of `github.com/radu-matei/kube-toolkit/example/scaffold` are replaced with `github.com/mumoshu/myproj`.